build-suggestions/4.19.yaml

```diff
@@ -1,5 +1,5 @@
 default:
-  minor_min: 4.18.6
+  minor_min: 4.18.11
```

Comment:
just a note in case this gets reverted.
We also rely on this for https://issues.redhat.com/browse/MON-4205
please reach out to us if you want to revert this.